src/main/java/Eraser.java

```diff
@@ -15,30 +15,33 @@ public Eraser(int durability) {
     }
 
     public void erase(Paper paper, String textToErase) {
-        String text = paper.getText();
+        int lastOccurrenceOfTextToErase = paper.getText().lastIndexOf(textToErase);
```

Comment:
I'm back and forth about if inlining `paper.getText()` is a good idea, but it _seemed_ to make things a little more concise, even with calling the getter more often...